lib/api/protocol/getContext.js

```diff
@@ -0,0 +1,35 @@
+const ProtocolAction = require('./_base-action.js');
+
+/**
+ * Get the current context in which Appium is running. Used when testing hybrid mobile apps using Appium.
+ *
+ * More info here: https://appium.io/docs/en/commands/context/get-context/
+ *
+ * @example
+ * module.exports = {
+ *   'get current context': function (browser) {
```

Comment:
Should we use `client` variable name instead of `browser` for mobile-specific commands?

Reply:
Yes